lib/pharos/phases/pull_master_images.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+
+module Pharos
+  module Phases
+    class PullMasterImages < Pharos::Phase
+      title "Pull control plane images"
+
+      def call
+        logger.info { "Pulling control plane images ..." }
+        cfg = kubeadm.generate_yaml_config
+        ssh.tempfile(content: cfg, prefix: "kubeadm.cfg") do |tmp_file|
+          @host.ssh.exec!("sudo kubeadm config images pull --config #{tmp_file}")
+        end
+      end
+
+      def kubeadm
```

Comment:
Should maybe be called something else than "kubeadm" as it returns a config generator

Reply:
Maybe... but this is c&p from another phase.. does it make this better/worse? 🙈